tests/unit-tests/mocks/gdblib.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import types
+from unittest.mock import MagicMock
```

Comment:
Do you think mixing `unittest` and `pytest` will be okay?

Reply:
Yup, pytest is designed to work well with `unittest`